kitsune/messages/forms.py

```diff
@@ -1,23 +1,43 @@
 from django import forms
 from django.utils.translation import gettext_lazy as _lazy
 
-from kitsune.sumo.form_fields import MultiUsernameField
-
-
-TO_PLACEHOLDER = _lazy("username1, username2,...")
+from kitsune.sumo.form_fields import MultiGroupField, MultiUsernameField
 
 
 class MessageForm(forms.Form):
     """Form send a private message."""
 
-    to = MultiUsernameField(
-        label=_lazy("To:"),
-        widget=forms.TextInput(
-            attrs={"placeholder": TO_PLACEHOLDER, "class": "user-autocomplete"}
-        ),
-    )
-    message = forms.CharField(label=_lazy("Message:"), max_length=10000, widget=forms.Textarea)
-    in_reply_to = forms.IntegerField(widget=forms.HiddenInput, required=False)
+    def __init__(self, *args, **kwargs):
+        self.user = kwargs.pop("user", None)
+        super(MessageForm, self).__init__(*args, **kwargs)
+
+        self.fields["to"] = MultiUsernameField(
```

Comment:
Let's move all the field declarations outside of the `__init__`

Reply:
Regarding field ordering, you can also use `field_order`
https://docs.djangoproject.com/en/5.0/ref/forms/api/#django.forms.Form.field_order

Reply:
I wonder if we could combine `to` and `to_group` into a single field and add autocompletion for groups only if the user is staff. It's going to be simpler for the user and the validation will be more straight-forward.